sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/AdoDtoHelper.java

```diff
@@ -177,6 +177,11 @@ protected int handlePullResponse(final boolean markAsRead, final AbstractAdoDao<
 
 	public int handlePulledList(AbstractAdoDao<ADO> dao, List<DTO> result)
 		throws DaoException, NoConnectionException, ServerConnectionException, ServerCommunicationException {
+
+		if (result == null) {
```

Comment:
Let's add a comment to the method that mentions that it returns the number handled results.